jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -711,24 +711,39 @@ def getLambdaARN (String stackName) {
 	def ARN = "";
 
 	try {
-		def cloudformation_resources = "";
-		cloudformation_resources = sh(returnStdout: true, script: "aws cloudformation describe-stacks --output json --stack-name ${stackName} --profile cloud-api")
-
-	    def parsedObject = parseJson(cloudformation_resources);
-		def outputs = parsedObject.Stacks[0].Outputs;
-
-		for (output in outputs) {
-			if (output.OutputKey == "HandlerLambdaFunctionQualifiedArn") {
-				ARN = output.OutputValue
+		ARN = tryDescribeStacks(stackName);
+	} catch (ex) {
+		
+		if (ex.getMessage().contains("The security token included in the request is invalid")) {
```

Comment:
this does not work in replay - as per documentation contains is part of toString method in groovy. @SiniWilson do you know?